src/include/daos_srv/vea.h

```diff
@@ -330,4 +331,12 @@ void *vea_metrics_alloc(const char *path, int tgt_id);
  */
 int vea_metrics_count(void);
 
+/**
+ * Set the VEA metric defining the NVME usage.
+ *
+ * \param metrics   [IN|OUT]	Metrics to updated
+ * \param nvme_used [IN]	Metric to be set
```

Comment:
I think this is the correct format based on other usages
```suggestion
 * \param[in,out] metrics  	Metrics to updated
 * \param[in] nvme_used [IN]	Metric to be set
```

Reply:
I agree that you propose is the one which should probably be used for being doxygen compliant:
https://www.doxygen.nl/manual/commands.html#cmdparam
However, the Upper case one that I used is the one which seems to be used for the description of all the C DAOS functions:
- https://github.com/daos-stack/daos/blob/master/src/include/daos_srv/vea.h
- https://github.com/daos-stack/daos/blob/f45b589bd0467e2fa6d8a11ed4068c32ecbcfa36/src/include/daos/mem.h#L230

Thus, I would prefer to stick to the current C code autodoc convention.